modules/base/CMakeLists.txt

```diff
@@ -261,7 +261,9 @@ ivw_add_unittest(${TEST_FILES})
 
 #--------------------------------------------------------------------
 # Create module
-ivw_create_module(${SOURCE_FILES} ${MOC_FILES} ${HEADER_FILES})
+ivw_create_module(NO_PCH ${SOURCE_FILES} ${MOC_FILES} ${HEADER_FILES})
```

Comment:
why?

Reply:
It is turned on by the `ivw_target_pch_headers`-call below.

Reply:
I've done some minor changes so I can probably remove it, investigating.

Reply:
Could not remove it, the call to `ivw_target_pch_headers` will then result in cmake error `PRECOMPILE_HEADERS_REUSE_FROM property is already set on target`